src/lib/Select/Select.svelte

```diff
@@ -216,8 +223,8 @@
   }
 
   .arrow {
-    height: 16px;
-    width: 16px;
+    height:var(--arrow-height, 16px);
```

Comment:
rename to dropdown-icon-height and width

Reply:
done